pkg/server/startup.go

```diff
@@ -56,6 +65,46 @@ func bootstrapResources(rioContext *types.Context, systemNamespace string) error
 	return controllerStack.Deploy(answer)
 }
 
+func configureFeature(rioContext *types.Context, systemNamespace string) error {
+	conf, err := config.GetConfig(systemNamespace, rioContext.Core.Core().V1().ConfigMap())
+	if err != nil {
+		return err
+	}
+
+	if conf.Features == nil {
+		conf.Features = map[string]config.FeatureConfig{}
+	}
+	if config.ConfigController.Features == "*" {
+		f := conf.Features["*"]
+		f.Enabled = &[]bool{true}[0]
+		conf.Features["*"] = f
+	} else {
+		for _, feature := range strings.Split(config.ConfigController.Features, ",") {
+			if feature == "" {
+				continue
+			}
+			f := conf.Features[feature]
+			f.Enabled = &[]bool{true}[0]
+			conf.Features[feature] = f
```

Comment:
Why can't you just do `conf.Features[feature].Enabled =...` ?

Reply:
I don't think golang allows you to do that.